codex-ui/src/vue/components/context-menu/ContextMenu.vue

```diff
@@ -5,22 +5,29 @@
       :class="$style['context-menu__search']"
     >
       <Input
+        v-model="searchTerm"
         icon="Search"
         placeholder="Search"
       />
       <ContextMenuItem :item="separator" />
     </div>
     <div :class="$style['context-menu__scrollable']">
-      <ContextMenuItem
-        v-for="(item, index) in items"
-        :key="index"
-        :item="item"
-      />
+      <template v-if="filteredItems.length > 0">
+        <ContextMenuItem
+          v-for="(item, index) in filteredItems"
+          :key="index"
+          :item="item"
+        />
+      </template>
+      <template v-else>
+        <ContextMenuItem :item="messageItem" />
+      </template>
```

Comment:
(*) comment:
if we do not filter items with type: message, we will have two cases
- items are only with type message element
```
{ type: 'message'}
``` 
- some items without type message